jamventProto/jamctrllib/JamCtrlHw.cpp

```diff
@@ -0,0 +1,28 @@
+
```

Comment:
Carly, Great start.  Good for us to get practice with GitHub branches, pull requests and reviews.  I have not really used GitHub much.